Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -457,7 +457,7 @@ end
 
 # Tier up immediately, while saving full vectors in argument FPRs
 macro wasmPrologueSIMD()
-if WEBASSEMBLY_OMGJIT and not ARMv7
+if (WEBASSEMBLY_BBQJIT or WEBASSEMBLY_OMGJIT) and not ARMv7
```

Comment:
Should the SIMD prologue still be disabled for ARMv7?

Reply:
We currently don't support any SIMD on ARMv7, so I think yes?